### ISSUE TITLE
Create hex logo

### ISSUE THREAD

**Issue by peterdesmet** [adokter/bioRad#137]
I want to create a hex logo for the bioRad package. [Hex logos](http://hexb.in/) are becoming a bit of a norm for R packages. Once we have one, we can order stickers and decorate our laptops. ✨💻✨

Here's a first attempt.

![logo_green](https://user-images.githubusercontent.com/600993/45691836-aa648d00-bb59-11e8-9ba2-95540b8d9131.png)
![logo_purple](https://user-images.githubusercontent.com/600993/45691837-aa648d00-bb59-11e8-80ed-00c0ab0f0b80.png)

Todo:

- [x] Use better background plot (@adokter)
- [x] Decide between ENRAM green, ENRAM purple or another color. I think it would be nice to keep ENRAM a little bit alive via the logo color. 😌@adokter @plieper @CeciliaNilsson709 preferences?
- [x] Add URL along right bottom line: which one do we use: https://adokter.github.io/bioRad or https://github.com/adokter/bioRad. The first one is nicer, the latter one will always redirect to the correct repo (if we would move it). -> no URL

**Comment by adokter** [comment]
I'm thinking we might have to keep it very simple for a logo.

How about a typical bloom, or maybe even a layering event like in the bioRad paper

[KBRO20170514_plot_dbz.pdf](https://github.com/adokter/bioRad/files/2393368/KBRO20170514_plot_dbz.pdf)
![kbro20170514_plot_dbz](https://user-images.githubusercontent.com/8268415/45698348-c6eece00-bb35-11e8-94c9-aff48854d2e4.png)

**Comment by adokter** [comment]
Of the two I like green as a color better

**Comment by peterdesmet** [comment]
Nice, do you have that bloom without the gray grid background (just white)? Or the pvol file and code to recreate it?

**Comment by CeciliaNilsson709** [comment]
Nice!

I like the purple one with the green blooms in it  😬


>  Once we have one, we can order stickers and decorate our laptops. ✨💻✨

mhm...

**Comment by CeciliaNilsson709** [comment]
They look great! I don't think anyone will notice that the bloom is cut off. After consulting an expert @adokter and I have now agreed on the purple one too 😌

**Comment by adokter** [comment]
pvol file here:
https://www.dropbox.com/s/2q9l70jn6pymm2u/KBRO20170514_055831_V06?dl=1

Code to make the figure:
```
pvol=read_pvolfile("KBRO20170514_055831_V06")
my_ppi=project_as_ppi(pvol$scans[[2]],range_max=120000)
p_dbz=plot(my_ppi,xlim=c(-100000,100000),ylim=c(-100000,100000),zlim=c(-10,20))
ggsave("KBRO20170514_plot_dbz.pdf",p_dbz,scale=.75)
```

**Comment by peterdesmet** [comment]
@adokter does that code require a running docker?

**Comment by adokter** [comment]
> @adokter does that code require a running docker?

Yes you do ... is that a problem?

**Comment by peterdesmet** [comment]
Didn't realize `read_pvolfile` required docker and never had to use it until now. 😬Got it installed and working now. Updated code for plot:

```R
library(bioRad)
library(ggplot2)
pvol <- read_pvolfile("KBRO20170514_055831_V06")
my_ppi <- project_as_ppi(pvol$scans[[2]], range_max = 120000)

plot <- plot(my_ppi, xlim = c(-120000,120000), ylim = c(-120000,120000), zlim = c(-10,20)) + 
          theme_void() + 
          theme(legend.position = "none")
          
ggsave("bloom.png", plot, bg = "transparent", units = "cm", width = 10, height = 10, scale = .75)
```

**Comment by peterdesmet** [comment]
Result:

![logo_biorad](https://user-images.githubusercontent.com/600993/45869509-ecbfe100-bd88-11e8-9ee8-8e48fcf76c78.png)

**Comment by peterdesmet** [comment]
@adokter @stijnvanhoey @CeciliaNilsson709 ❤️Awesome! Logo included in repo + added to website (pending #143).

Here's the 300dpi hres version:

![biorad-hexagon-1 73x2](https://user-images.githubusercontent.com/600993/45917720-19085a00-be7c-11e8-8853-3a338b9ca076.png)

Now we only need to [order stickers](http://hexb.in/sticker.html). 🤩

**Comment by plieper** [comment]
Very nice, guys! ;-)
